### PR TITLE
move special collections to new database

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

**Comment by acozine** [comment]
Related to #2737.

**Comment by acozine** [comment]
Related to #2737.

File: playbooks/special_collections.yml [hunk 1]
```diff
@@ -10,17 +13,17 @@
         - run a cap for special_collections
         - where needed make sure server is greenlisted on [lib-ponyexpress](https://github.com/pulibrary/pul-the-hard-way/blob/master/services/smtp-mail-server.md)
   vars_files:
-    - ../group_vars/special_collections/special_collections_prod.yml
+    - ../group_vars/special_collections/{{ runtime_env | default('staging') }}.yml
     - ../group_vars/special_collections/vault.yml
   roles:
     - role: datadog
```
Comment (acozine) [inline playbooks/special_collections.yml:19]: Only run the datadog role on production servers.

**Review by acozine — COMMENTED** [review]
This mixes in changes to EZProxy and changes to Special Collections. Is that intentional? I found one other suggestion . . . otherwise looks good.

File: playbooks/special_collections.yml (hunk 1 above)
Comment (kayiwa) [reply playbooks/special_collections.yml:19]: the CI breaks without ezproxy in it. So I cherry picked it from main.

**Review by carolyncole — APPROVED** [review]
Checked the deployment. Everything seems good.

**Pull request merged**